### PR TITLE
62 split user pw into extra table

### PR DESCRIPTION
**Pull request by tonioriol**

### CODE REVIEW

File: prisma/seed.ts
Comment (tonioriol) [inline prisma/seed.ts:1]: that can be ignored. Its just the seeds file to seed data for testing.

File: src/lib/components/ShareChatModal.svelte
Comment (tonioriol) [inline src/lib/components/ShareChatModal.svelte:1]: unrelated, just some extras that i've found to fix

File: src/lib/components/ChatRoom.svelte
Comment (tonioriol) [inline src/lib/components/ChatRoom.svelte:1]: unrelated, just some extras that i've found to fix

File: src/lib/components/Sidebar.svelte
Comment (tonioriol) [inline src/lib/components/Sidebar.svelte:1]: unrelated, just some extras that i've found to fix

File: src/lib/server/utils/database.ts
Comment (tonioriol) [inline src/lib/server/utils/database.ts:1]: moved all that to the user, and refactored to use database queries straight away. Which i faster.

**Review by brokelyn — APPROVED** [review]
Just briefly check this and it looks good :+1: Very need change :D

**Pull request merged**